dashboard/app/views/levels/_lab2.html.haml

```diff
@@ -1,5 +1,4 @@
-#codeApp{style: "position: absolute; top: 60px; bottom: 10px; left: 10px; right: 10px"}
```

Comment:
there's a bunch of styles in `application.scss` (and other places) that are keyed on `#codeApp`, do we not need those?

Reply:
Yeah as far as I can tell none of these are really affected/used by Lab2? Since the Lab2 container uses `position: fixed`, it seems to ignore all the margin and offset applied to `#codeApp` and the other usages seem to be lab-specific (like Applab, oceans, jigsaw, etc). StudioApp/code-studio.js do some of manipulation with the #codeApp element but again shouldn't affect Lab2 since we don't use those classes.

Reply:
If we don't use the styles seems good to remove it!